packages/fxa-auth-server/lib/routes/security-events.js

```diff
@@ -0,0 +1,31 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict';
+
+const error = require('../error');
+const isA = require('joi');
+
+module.exports = (log, db, config) => {
+  return [
+    {
+      method: 'GET',
+      path: '/securityEvents/:id',
```

Comment:
You will need to put this behind a `sessionToken` authentication strategy, https://github.com/mozilla/fxa/blob/b22d8cdeb47cf4dcd52aa90fc8e21f4eedd24725/packages/fxa-auth-server/lib/routes/emails.js#L36. When you do that, you will have access to the the `uid` from the [credentials object](https://github.com/mozilla/fxa/blob/b22d8cdeb47cf4dcd52aa90fc8e21f4eedd24725/packages/fxa-auth-server/lib/routes/emails.js#L60).

You will also be able to remove the `/:id` part from the path.